packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -25,48 +25,68 @@ const propTypes = {
    */
   accessoryAlignment: PropTypes.oneOf(['alignTop', 'alignCenter']),
   /**
+   * The height and width value the accessory should receive. Maximum scalar is 60px.
+   */
+  accessoryScale: PropTypes.string,
+  /**
    * The react element to be placed in the start aligned accessory position.
    */
-  startAccessory: PropTypes.element,
+  startAccessory: PropTypes.node,
+  /**
+   * Indicates whether or not the allow space for the start accessory if none is given.
```

Comment:
I think you mean "to allow space" vs. "the allow space"

Same thing below for alignEndAccessory.